src/frontend_workspaces/agentic_chat/src/ConfigModal.css

```
@@ -440,6 +449,123 @@
+  min-width: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+}
+
+.knowledge-reindex-item__filename {
+  font-size: 13px;
+  line-height: 1.4;
+  color: var(--cds-text-primary, #161616);
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)